RDSSurveyor/src/eu/jacquet80/rds/input/GnsGroupReader.java

```diff
@@ -593,9 +593,9 @@ public void run() {
 								}
 								/*
 								 * TODO not sure if conversion is correct
-								 * (maximum is 0xFF in theory, 0x1F is maximum observed so far)
+								 * (maximum is 0xFF in theory, presumably 0x3F or more, up to 0x2C observed so far)
 								 */
-								gnsData.rssi = intData[6] * 0x842;
+								gnsData.rssi = intData[6] * 0x410;
```

Comment:
I suggest writing this as 0xFFFF / 0x3F. It's clearer and the compiler will fold constants.